src/win/process.c

```diff
@@ -1040,7 +1040,8 @@ int uv_spawn(uv_loop_t* loop,
     }
   }
 
-  err = uv__stdio_create(loop, options, &process->child_stdio_buffer);
+  child_stdio_buffer = NULL;
```

Comment:
This needs to be moved to before the first `goto done`, before line 972 if I count correctly.

Reply:
Thanks for catching this up, I was a bit careless here.